srtcore/core.cpp

```diff
@@ -8379,6 +8386,28 @@ void CUDT::processCtrlAck(const CPacket &ctrlpkt, const steady_clock::time_point
     leaveCS(m_StatsLock);
 }
 
+int CUDT::processLossSeq(const int seqno, const time_point& time_nak, const time_point& time_now)
```

Comment:
1. I think you should embrace `time_now` in `IF_HEAVY_LOGGING` to avoid compiler warning when compiling in release mode.

2. This function seems to be unused, what is this?